gulp/tasks/serve.js

```diff
@@ -28,10 +29,10 @@ gulp.task('serve', ['prepare'], () => {
         return pathname;
     };
 
-    const watch = (pathname, tasks) => gulp.watch(sanitize(pathname), tasks);
+    const watch = (pathname, tasks) => gwatch(sanitize(pathname), () => runSequence(...tasks));
```

Comment:
I think with `gulp-watch` we don’t need `sanitize` function anymore. I tested it without it and it works fine for both array and string globs (modify, create, delete).

```js
const watch = (glob, tasks) => gwatch(glob, () => runSequence(...tasks));
```